src/Microsoft.ML.Transforms/Text/WrappedTextTransformers.cs

```diff
@@ -10,11 +10,30 @@
 
 namespace Microsoft.ML.Transforms.Text
 {
-
     /// <summary>
-    /// Produces a bag of counts of ngrams (sequences of consecutive words) in a given text.
-    /// It does so by building a dictionary of ngrams and using the id in the dictionary as the index in the bag.
+    /// <see cref="IEstimator{TTransformer}"/> for the <see cref="ITransformer"/>.
     /// </summary>
+    /// <remarks>
+    /// <format type="text/markdown"><![CDATA[
+    /// ###  Estimator Characteristics
+    /// |  |  |
+    /// | -- | -- |
+    /// | Does this estimator need to look at the data to train its parameters? | Yes |
+    /// | Input column data type | Vector of [Text](<xref:Microsoft.ML.Data.TextDataViewType>) |
+    /// | Output column data type | Vector of known-size of <xref:System.Single> |
+    ///
+    /// The resulting <xref:Microsoft.ML.ITransformer/> creates a new column, named as specified in the output column name parameters, and
+    /// produces a vector of counts of n-grams (sequences of consecutive words of length 1-n) from a given data.
```

Comment:
>sequences of consecutive words of length 1-n [](start = 48, length = 44)

n-gram = 'sequences of n consecutive words'

Reply:
I understand you want me to change this, I just don't understand your proposal

---
In reply to: [277088838](https://github.com/dotnet/machinelearning/pull/3440#discussion_r277088838) [](ancestors = 277088838)